app/controllers/internal/feedback_messages_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
       includes(:reporter, :notes).
       order("feedback_messages.created_at DESC").
       page(params[:page] || 1).per(5)
-    @email_messages = EmailMessage.find_for_reports(@feedback_messages.pluck(:id))
+    @email_messages = EmailMessage.find_for_reports(@feedback_messages)
```

Comment:
`EmailMessage.find_for_reports` uses feedback messages IDs for a query.

If we don't use `pluck` but pass the object themselves Rails is going to issue only one query using a subselect. something like:

`SELECT * FROM email_messages WHERE feedback_message_id IN (SELECT id FROM feedback_messages)` instead of two separate queries